include/config/config_camera.h

```diff
@@ -44,6 +44,16 @@
  */
 #define PARALLEL_LAKITU_CAM
 
+/**
+ * These defines decide whether the turning, error and zoom out/zoom in out sounds are enabled, and which sounds to play if enabled.
+ * The turning, zooming out and zooming in sounds have been replaced with less annoying ones. 
+ * If you want the vanilla ones back, set them to SOUND_MENU_CAMERA_TURN, SOUND_MENU_CAMERA_ZOOM_OUT and SOUND_MENU_CAMERA_ZOOM_IN respectively.
```

Comment:
i think this should have a safeguard so that if these values are commented out, it automatically sets them to vanilla (instead of having the vanilla sounds in a comment here)

Reply:
I agree, commenting these out should default to vanilla. I still think these should be disabled by default (uncommenting these is super easy after all), but if people really want this to be default, then the description for sure needs updated to reflect that. (Just replace the last line I guess since that'll become sorta redundant.)

Reply:
> I agree, commenting these out should default to vanilla. I still think these should be disabled by default (uncommenting these is super easy after all), but if people really want this to be default, then the description for sure needs updated to reflect that. (Just replace the last line I guess since that'll become sorta redundant.)

we discussed this in the discord but i never followed up on github - iirc the plan was to make commenting the define make no sound at all play, and if it's defined but left empty, *then* it plays the vanilla sound

Reply:
Actually wait, how would you implement this? I'm not familiar with whether you can differentiate an empty define from one with a value of many multiple possibilities.